.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+sudo: false
+dist: trusty
+python:
+    - "2.7"
+install:
+    - pip install zc.buildout
+    - buildout bootstrap
```

Comment:
I actually like that way of bootstrapping. It might circumvent the issue of deprecated and old `bootstrap.py` in all these buildout based repositories.